scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1187,7 +1185,7 @@ private Boolean isExecutionCalendarGIDefined(String jobDescriptor) {
                         exists = true;
                     }
                     if (isAttributeExecCalendarValueDefined(attribute, "value") && exists) {
-                        if (!attribute.getNodeValue().isEmpty() && !attribute.getNodeValue().isEmpty() && exists) {
+                        if (!attribute.getNodeValue().isEmpty() && exists) {
```

Comment:
Just a question plz, can the nodeValue be null (I'm asking because you removed the null check, and the stock .isEmpty() doesn't perform null check, so it could fail if you get a null)

Reply:
I removed it because it was a replicate of the same check in the same if clause.